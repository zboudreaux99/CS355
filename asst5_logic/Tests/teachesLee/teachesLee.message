Output line 0:
  <i>expected: </i> Jullanar Woodbury, Jeffrey Yankanwich, Jonathon Yeatts, Albert <span class='expected'>no</span> --- end of output --- 
  <i>observed: </i>Jullanar Woodbury, Jeffrey Yankanwich, Jonathon Yeatts, Albert -<span class='observed'>-</span>- end of output --- 
