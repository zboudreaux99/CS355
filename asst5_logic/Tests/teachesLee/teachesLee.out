Adapala, Renate
Al-douli, Allen
Althouse, Glenn
Barton, Gustavo
Belur, Shirley
Burgess, Juanda
Burr, Jared
Butler, Julie
Casey, Catherine
Diggs, Tierra
Dobrydnev, Dorothy
Dye, Xin
Ejike, Agustin
Farag, Jeff
Finman, Richelle
Garrett, Gustavo
Gary, Brett
Gibbs, Yohannes
Gills, Bangfei
Givens, Kathy
Goldstein, Courtney
Gopal, Jillian
Gorozabel, Luz
Grosz, Frank
Hall, Hamad
Hampton, Heather
Hetland, Kyle
Hong, Gholam
Huff, Lakesha
Irintchev, Chihyun
Irintchev, Jr.
Junio, Tamas
Kelly, Christopher
Lenda, Rahmi
Linthicum, Chien-liang
Linthicum, Ralph
Little, Zubair
Lopez, Tariq
Major, Balaje
Malkin, Derek
Meinertzhagen, Wei
Merrill, Brian
Mervicker, Christin
Mills, Adrian
Mills, Kerry
Morrisette, Rhonda
O Reilly Chunxiang
OReilly, Chunxiang
Ost, Xiaobing
Page, Sumant
Patel, Sarah-jean
Paul, Catherine
Pham, Gerard
Potluri, Catherine
Query, Tameishaon
Ramirez, Candias
Randolph, Darrah
Riley, Mohsina
Robertson, Karen
Russell, Jongsoon
Shi, Nichelle
Stoner, Natia
Suggs, Christine
Svetahor, Sultana
Swao, Barbara
Taxis, Samuel
Thymides, Glen
Tirol, Kyra
Tomioka, Gerhard
Waldecker, Arthur
Williams, Keith
Woltz, Jullanar
Woodbury, Jeffrey
Yankanwich, Jonathon
Yeatts, Albert
